data/catalogue/training_dataset_ratios.json

```diff
@@ -0,0 +1,198 @@
+[
```

Comment:
ratios were basically computed by:
 - getting the number of tokens per dataset
 - computing the probability by normalising by the total amount of tokens.
 
 Hopefully this means it follows the datasets distribution.